pkg/agent/manager/cache/cache.go

```diff
@@ -177,7 +182,7 @@ func (c *Cache) SubscribeToWorkloadUpdates(selectors []*common.Selector) Subscri
 	return sub
 }
 
-func (c *Cache) UpdateEntries(update *UpdateEntries, checkSVID func(*common.RegistrationEntry, *common.RegistrationEntry, *X509SVID)) {
+func (c *Cache) UpdateEntries(update *UpdateEntries, checkSVID func(*common.RegistrationEntry, *common.RegistrationEntry, *X509SVID) bool) {
```

Comment:
A code comment here on the meaning of the `bool` return value for the required func would be good. Probably worthwhile describing the entire func and its expected behavior.